### PR TITLE
Created Scripting file. successfully deployed to anvil local network.…

### PR DESCRIPTION
**Pull request by miguelmendez1208**
… will add more complex interactions later. maybe its best to just copy the entirety of ClExecutor.t.sol to the scripting file? unsure. will find out later

### CODE REVIEW

File: contracts/README.md [hunk 1]
```diff
@@ -64,3 +64,37 @@ $ forge --help
...
+        mid = new Mid();
+        wide = new Wide();
+        ramsesV2Pool = new RamsesV2Pool();
+        gaugeV2 = new GaugeV2();
+        votingEscrow = new VotingEscrow();
+        clExecutor = new ClExecutor(
+
+private key:
+        
+0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```
Comment (xrave110) [inline contracts/README.md]: Why is there private key in the README ?

**Review by xrave110 — CHANGES_REQUESTED** [review]
README with private key should not be pushed

File: contracts/README.md (hunk 1 above)
Comment (miguelmendez1208) [reply contracts/README.md]: its a test private key

**Review by miguelmendez1208 — COMMENTED** [review]
anvil test private key removed from readme

**Review by xrave110 — APPROVED** [review]
README with private key should not be pushed

**Pull request merged**